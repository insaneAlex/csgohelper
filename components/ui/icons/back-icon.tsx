import React, {FC} from "react";
import backIcon from "./back-svgrepo-com.svg";
import Image from "next/image";

export const BackIcon: FC = () => {
  return <Image priority src={backIcon} height={32} width={32} alt="Back" />;
};
