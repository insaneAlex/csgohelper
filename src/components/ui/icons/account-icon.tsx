import React, {FC} from 'react';

type Props = {width?: number; height?: number; fill?: string};

export const AccountIcon: FC<Props> = ({fill = 'gainsboro', width = 16, height = 16}) => (
  <svg data-testid="AccountIcon" width={width} height={height} viewBox="0 0 227 240.5">
    <path
      d="m9.54,239.64c-4.71,0-8.54-3.83-8.54-8.54v-76.8c0-26.79,15.93-51.02,40.58-61.72l.86-.37-.78-.51c-14.14-9.14-22.58-24.59-22.58-41.31C19.07,23.27,41.14,1.2,68.26,1.2s49.19,22.07,49.19,49.19c0,16.73-8.44,32.17-22.58,41.31l-.78.51.86.37c14.98,6.5,27.09,18.26,34.11,33.11l.35.73.5-.64c4.23-5.4,9.38-9.89,15.3-13.35l.59-.34-.5-.46c-8.56-7.77-13.28-18.29-13.28-29.62,0-22.14,18.01-40.16,40.16-40.16s40.16,18.02,40.16,40.16c0,11.33-4.72,21.85-13.28,29.62l-.5.46.59.34c16.5,9.63,26.75,27.4,26.75,46.38v72.28c0,4.71-3.83,8.54-8.54,8.54H9.54Zm162.63-117.46c-20.2,0-36.64,16.44-36.64,36.64v63.75h73.28v-63.75c0-20.2-16.44-36.64-36.64-36.64Zm-103.91-18.07c-27.68,0-50.19,22.52-50.19,50.19v68.26h100.39v-68.26c0-27.68-22.52-50.19-50.19-50.19Zm103.91-45.18c-12.73,0-23.09,10.36-23.09,23.09s10.36,23.09,23.09,23.09,23.09-10.36,23.09-23.09-10.36-23.09-23.09-23.09ZM68.26,18.27c-17.71,0-32.12,14.41-32.12,32.12s14.41,32.12,32.12,32.12,32.12-14.41,32.12-32.12-14.41-32.12-32.12-32.12Z"
      fill={fill}
    />
    <path d="m68.26,1.7c26.85,0,48.69,21.84,48.69,48.69,0,16.55-8.36,31.84-22.36,40.89l-1.57,1.01,1.71.74c14.87,6.45,26.89,18.12,33.86,32.87l.69,1.47,1-1.28c4.19-5.35,9.29-9.8,15.16-13.23l1.18-.69-1.01-.92c-8.46-7.68-13.11-18.07-13.11-29.25,0-21.87,17.79-39.66,39.66-39.66s39.66,17.79,39.66,39.66c0,11.18-4.66,21.57-13.11,29.25l-1.01.92,1.18.69c16.35,9.54,26.5,27.14,26.5,45.95v72.28c0,4.43-3.6,8.04-8.04,8.04H9.53c-4.43,0-8.04-3.6-8.04-8.04v-76.8c0-26.59,15.81-50.64,40.28-61.26l1.71-.74-1.57-1.01c-14-9.05-22.36-24.34-22.36-40.89C19.57,23.54,41.41,1.7,68.26,1.7m0,81.32c17.99,0,32.62-14.63,32.62-32.62s-14.63-32.62-32.62-32.62-32.62,14.63-32.62,32.62,14.63,32.62,32.62,32.62m103.91,22.59c13.01,0,23.59-10.58,23.59-23.59s-10.58-23.59-23.59-23.59-23.59,10.58-23.59,23.59,10.58,23.59,23.59,23.59M17.57,223.07h101.39v-68.76c0-27.95-22.74-50.69-50.69-50.69s-50.69,22.74-50.69,50.69v68.76m117.46,0h74.28v-64.25c0-20.48-16.66-37.14-37.14-37.14s-37.14,16.66-37.14,37.14v64.25M68.26.7C40.87.7,18.57,23,18.57,50.39c0,17.49,9.11,32.87,22.81,41.73C17.36,102.55.5,126.48.5,154.3v76.8c0,4.99,4.04,9.04,9.04,9.04h207.81c4.99,0,9.04-4.04,9.04-9.04v-72.28c0-19.97-10.88-37.4-27-46.81,8.2-7.44,13.44-18.07,13.44-29.99,0-22.42-18.24-40.66-40.66-40.66s-40.66,18.24-40.66,40.66c0,11.92,5.25,22.55,13.44,29.99-5.97,3.49-11.21,8.06-15.45,13.47-7.02-14.86-19.24-26.8-34.36-33.36,13.7-8.86,22.81-24.24,22.81-41.73C117.96,23,95.66.7,68.26.7h0Zm0,81.32c-17.44,0-31.62-14.19-31.62-31.62s14.19-31.62,31.62-31.62,31.62,14.19,31.62,31.62-14.19,31.62-31.62,31.62h0Zm103.91,22.59c-12.46,0-22.59-10.13-22.59-22.59s10.13-22.59,22.59-22.59,22.59,10.13,22.59,22.59-10.13,22.59-22.59,22.59h0ZM18.57,222.07v-67.76c0-27.4,22.3-49.69,49.69-49.69s49.69,22.3,49.69,49.69v67.76H18.57Zm117.46,0v-63.25c0-19.93,16.21-36.14,36.14-36.14s36.14,16.21,36.14,36.14v63.25h-72.28Z" />
  </svg>
);
