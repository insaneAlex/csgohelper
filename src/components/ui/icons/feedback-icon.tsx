import React, {type FC} from 'react';

type Props = {width?: number; height?: number; fill?: string};

export const FeedbackIcon: FC<Props> = ({fill = 'gainsboro', width = 16, height = 16}) => {
  return (
    <svg data-testid="FeedbackIcon" width={width} height={height} viewBox="0 0 216 254.5">
      <path
        d="m9.05,253.41c-4.71,0-8.54-3.83-8.54-8.54V9.96C.52,5.25,4.35,1.42,9.05,1.42h135.53s.09.02.16.04c.1.03.22.06.36.07.77.05,1.5.19,2.22.44l.25.08c.18.06.37.12.54.2,1.03.48,1.85,1.02,2.5,1.67l63.25,63.25c.66.66,1.19,1.45,1.67,2.5.09.19.14.37.21.56l.08.23c.25.74.4,1.46.44,2.21,0,.14.04.27.07.36.02.06.04.13.04.16v171.67c0,4.71-3.83,8.54-8.54,8.54H9.05Zm8.54-17.07h181.71V81.74h-54.71c-4.71,0-8.54-3.83-8.54-8.54V18.5H17.59v217.85ZM153.12,64.67h34.11l-34.11-34.11v34.11Z"
        fill={fill}
      />
      <path d="m144.54,1.92s.04.01.06.02c.11.03.27.08.48.09.72.04,1.4.18,2.09.42l.25.08c.19.06.35.11.49.17.98.45,1.75.97,2.36,1.57l63.25,63.25c.61.61,1.11,1.36,1.57,2.36.07.15.12.31.18.5l.08.25c.24.7.37,1.38.42,2.09.01.19.06.35.09.46,0,.02.01.04.02.06v171.63c0,4.43-3.6,8.04-8.04,8.04H9.05c-4.43,0-8.04-3.6-8.04-8.04V9.96C1.02,5.53,4.62,1.92,9.05,1.92h135.49M17.09,236.84h182.71V81.24h-55.21c-4.43,0-8.04-3.6-8.04-8.04V17.99H17.09v218.85M152.62,65.17h35.81l-1.71-1.71-32.4-32.4-1.71-1.71v35.81M144.58.92H9.05C4.06.92.02,4.97.02,9.96v234.92c0,4.99,4.04,9.04,9.04,9.04h198.78c4.99,0,9.04-4.04,9.04-9.04V73.21c0-.19-.1-.37-.11-.56-.05-.8-.21-1.58-.47-2.34-.1-.29-.17-.56-.3-.84-.44-.96-.99-1.87-1.77-2.65L150.97,3.57c-.78-.78-1.69-1.33-2.65-1.77-.27-.12-.55-.2-.84-.3-.76-.26-1.54-.42-2.35-.47-.19-.01-.36-.11-.56-.11h0ZM18.09,235.84V18.99h117.46v54.21c0,4.99,4.04,9.04,9.04,9.04h54.21v153.6H18.09ZM153.62,64.17V31.77l32.4,32.4h-32.4Z" />
      <path
        d="m45.19,108.85c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
        fill={fill}
      />
      <path d="m171.69,92.28c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z" />
      <path
        d="m45.19,63.67c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h63.25c4.71,0,8.53,3.83,8.53,8.54s-3.83,8.54-8.53,8.54h-63.25Z"
        fill={fill}
      />
      <path d="m108.44,47.1c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04h-63.25c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h63.25m0-1h-63.25c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h63.25c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z" />
      <path
        d="m45.19,154.02c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
        fill={fill}
      />
      <path d="m171.69,137.45c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z" />
      <path
        d="m45.19,199.2c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
        fill={fill}
      />
      <path d="m171.69,182.63c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z" />
    </svg>
  );
};
