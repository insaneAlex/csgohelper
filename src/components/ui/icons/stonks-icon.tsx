import React, {type FC} from 'react';

type Props = {width?: number; height?: number; fill?: string};

export const StonksIcon: FC<Props> = ({fill = 'gainsboro', width = 16, height = 16}) => {
  return (
    <svg data-testid="StonksIcon" width={width} height={height} viewBox="0 0 243.95 250.24">
      <path
        d="m54.21,141.82H9.04c-4.99,0-9.04,4.04-9.04,9.04v90.35c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04v-90.35c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,90.35h-27.11v-72.28h27.11v72.28Z"
        fill={fill}
      />
      <path
        d="m144.56,96.64h-45.18c-4.99,0-9.04,4.04-9.04,9.04v135.53c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04V105.68c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,135.53h-27.11v-117.46h27.11v117.46Z"
        fill={fill}
      />
      <path
        d="m234.92,51.46h-45.18c-4.99,0-9.04,4.04-9.04,9.04v180.71c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04V60.5c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,180.71h-27.11V69.53h27.11v162.64Z"
        fill={fill}
      />
      <path
        d="m170.62,13.75c-.25-1-.72-1.91-1.27-2.75-.09-.14-.1-.3-.2-.44-.11-.16-.3-.23-.42-.38-.64-.75-1.33-1.46-2.18-1.99-1.03-.62-2.15-1.04-3.32-1.21L118.58.1c-4.91-.74-9.55,2.62-10.31,7.55-.76,4.93,2.62,9.55,7.55,10.31l23.09,3.56L22.55,106.85c-4.02,2.96-4.89,8.61-1.94,12.64,1.77,2.41,4.52,3.69,7.3,3.69,1.85,0,3.72-.56,5.34-1.75l116.36-85.33-3.55,23.08c-.76,4.93,2.62,9.55,7.55,10.31.47.07.94.11,1.39.11,4.39,0,8.23-3.19,8.92-7.66l6.86-44.65c.18-1.16.12-2.36-.16-3.53Z"
        fill={fill}
      />
    </svg>
  );
};
