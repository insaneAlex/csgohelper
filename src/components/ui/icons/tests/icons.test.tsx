import {HamburgerIcon} from '../hamburger-icon';
import {render} from '@testing-library/react';
import {FeedbackIcon} from '../feedback-icon';
import {AccountIcon} from '../account-icon';
import {StonksIcon} from '../stonks-icon';
import {BackIcon} from '../back-icon';

describe('AccountIcon', () => {
  it('should render correctly', () => {
    const {container} = render(<AccountIcon />);
    expect(container).toMatchSnapshot();
  });
});

describe('BackIcon', () => {
  it('should render correctly', () => {
    const {container} = render(<BackIcon />);
    expect(container).toMatchSnapshot();
  });
});
describe('FeedbackIcon', () => {
  it('should render correctly', () => {
    const {container} = render(<FeedbackIcon />);
    expect(container).toMatchSnapshot();
  });
});
describe('HamburgerIcon', () => {
  it('should render correctly', () => {
    const {container} = render(<HamburgerIcon />);
    expect(container).toMatchSnapshot();
  });
});
describe('StonksIcon', () => {
  it('should render correctly', () => {
    const {container} = render(<StonksIcon />);
    expect(container).toMatchSnapshot();
  });
});
