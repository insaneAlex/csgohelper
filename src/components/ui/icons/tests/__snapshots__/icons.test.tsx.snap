// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountIcon should render correctly 1`] = `
<div>
  <svg
    height="16"
    viewBox="0 0 227 240.5"
    width="16"
  >
    <path
      d="m9.54,239.64c-4.71,0-8.54-3.83-8.54-8.54v-76.8c0-26.79,15.93-51.02,40.58-61.72l.86-.37-.78-.51c-14.14-9.14-22.58-24.59-22.58-41.31C19.07,23.27,41.14,1.2,68.26,1.2s49.19,22.07,49.19,49.19c0,16.73-8.44,32.17-22.58,41.31l-.78.51.86.37c14.98,6.5,27.09,18.26,34.11,33.11l.35.73.5-.64c4.23-5.4,9.38-9.89,15.3-13.35l.59-.34-.5-.46c-8.56-7.77-13.28-18.29-13.28-29.62,0-22.14,18.01-40.16,40.16-40.16s40.16,18.02,40.16,40.16c0,11.33-4.72,21.85-13.28,29.62l-.5.46.59.34c16.5,9.63,26.75,27.4,26.75,46.38v72.28c0,4.71-3.83,8.54-8.54,8.54H9.54Zm162.63-117.46c-20.2,0-36.64,16.44-36.64,36.64v63.75h73.28v-63.75c0-20.2-16.44-36.64-36.64-36.64Zm-103.91-18.07c-27.68,0-50.19,22.52-50.19,50.19v68.26h100.39v-68.26c0-27.68-22.52-50.19-50.19-50.19Zm103.91-45.18c-12.73,0-23.09,10.36-23.09,23.09s10.36,23.09,23.09,23.09,23.09-10.36,23.09-23.09-10.36-23.09-23.09-23.09ZM68.26,18.27c-17.71,0-32.12,14.41-32.12,32.12s14.41,32.12,32.12,32.12,32.12-14.41,32.12-32.12-14.41-32.12-32.12-32.12Z"
      fill="gainsboro"
    />
    <path
      d="m68.26,1.7c26.85,0,48.69,21.84,48.69,48.69,0,16.55-8.36,31.84-22.36,40.89l-1.57,1.01,1.71.74c14.87,6.45,26.89,18.12,33.86,32.87l.69,1.47,1-1.28c4.19-5.35,9.29-9.8,15.16-13.23l1.18-.69-1.01-.92c-8.46-7.68-13.11-18.07-13.11-29.25,0-21.87,17.79-39.66,39.66-39.66s39.66,17.79,39.66,39.66c0,11.18-4.66,21.57-13.11,29.25l-1.01.92,1.18.69c16.35,9.54,26.5,27.14,26.5,45.95v72.28c0,4.43-3.6,8.04-8.04,8.04H9.53c-4.43,0-8.04-3.6-8.04-8.04v-76.8c0-26.59,15.81-50.64,40.28-61.26l1.71-.74-1.57-1.01c-14-9.05-22.36-24.34-22.36-40.89C19.57,23.54,41.41,1.7,68.26,1.7m0,81.32c17.99,0,32.62-14.63,32.62-32.62s-14.63-32.62-32.62-32.62-32.62,14.63-32.62,32.62,14.63,32.62,32.62,32.62m103.91,22.59c13.01,0,23.59-10.58,23.59-23.59s-10.58-23.59-23.59-23.59-23.59,10.58-23.59,23.59,10.58,23.59,23.59,23.59M17.57,223.07h101.39v-68.76c0-27.95-22.74-50.69-50.69-50.69s-50.69,22.74-50.69,50.69v68.76m117.46,0h74.28v-64.25c0-20.48-16.66-37.14-37.14-37.14s-37.14,16.66-37.14,37.14v64.25M68.26.7C40.87.7,18.57,23,18.57,50.39c0,17.49,9.11,32.87,22.81,41.73C17.36,102.55.5,126.48.5,154.3v76.8c0,4.99,4.04,9.04,9.04,9.04h207.81c4.99,0,9.04-4.04,9.04-9.04v-72.28c0-19.97-10.88-37.4-27-46.81,8.2-7.44,13.44-18.07,13.44-29.99,0-22.42-18.24-40.66-40.66-40.66s-40.66,18.24-40.66,40.66c0,11.92,5.25,22.55,13.44,29.99-5.97,3.49-11.21,8.06-15.45,13.47-7.02-14.86-19.24-26.8-34.36-33.36,13.7-8.86,22.81-24.24,22.81-41.73C117.96,23,95.66.7,68.26.7h0Zm0,81.32c-17.44,0-31.62-14.19-31.62-31.62s14.19-31.62,31.62-31.62,31.62,14.19,31.62,31.62-14.19,31.62-31.62,31.62h0Zm103.91,22.59c-12.46,0-22.59-10.13-22.59-22.59s10.13-22.59,22.59-22.59,22.59,10.13,22.59,22.59-10.13,22.59-22.59,22.59h0ZM18.57,222.07v-67.76c0-27.4,22.3-49.69,49.69-49.69s49.69,22.3,49.69,49.69v67.76H18.57Zm117.46,0v-63.25c0-19.93,16.21-36.14,36.14-36.14s36.14,16.21,36.14,36.14v63.25h-72.28Z"
    />
  </svg>
</div>
`;

exports[`BackIcon should render correctly 1`] = `
<div>
  <svg
    fill="#000000"
    height="16"
    viewBox="0 0 1024 1024"
    width="16"
  >
    <path
      d="M669.6 849.6c8.8 8 22.4 7.2 30.4-1.6s7.2-22.4-1.6-30.4l-309.6-280c-8-7.2-8-17.6 0-24.8l309.6-270.4c8.8-8 9.6-21.6 2.4-30.4-8-8.8-21.6-9.6-30.4-2.4L360.8 480.8c-27.2 24-28 64-0.8 88.8l309.6 280z"
      fill="gainsboro"
    />
  </svg>
</div>
`;

exports[`FeedbackIcon should render correctly 1`] = `
<div>
  <svg
    height="16"
    viewBox="0 0 216 254.5"
    width="16"
  >
    <path
      d="m9.05,253.41c-4.71,0-8.54-3.83-8.54-8.54V9.96C.52,5.25,4.35,1.42,9.05,1.42h135.53s.09.02.16.04c.1.03.22.06.36.07.77.05,1.5.19,2.22.44l.25.08c.18.06.37.12.54.2,1.03.48,1.85,1.02,2.5,1.67l63.25,63.25c.66.66,1.19,1.45,1.67,2.5.09.19.14.37.21.56l.08.23c.25.74.4,1.46.44,2.21,0,.14.04.27.07.36.02.06.04.13.04.16v171.67c0,4.71-3.83,8.54-8.54,8.54H9.05Zm8.54-17.07h181.71V81.74h-54.71c-4.71,0-8.54-3.83-8.54-8.54V18.5H17.59v217.85ZM153.12,64.67h34.11l-34.11-34.11v34.11Z"
      fill="gainsboro"
    />
    <path
      d="m144.54,1.92s.04.01.06.02c.11.03.27.08.48.09.72.04,1.4.18,2.09.42l.25.08c.19.06.35.11.49.17.98.45,1.75.97,2.36,1.57l63.25,63.25c.61.61,1.11,1.36,1.57,2.36.07.15.12.31.18.5l.08.25c.24.7.37,1.38.42,2.09.01.19.06.35.09.46,0,.02.01.04.02.06v171.63c0,4.43-3.6,8.04-8.04,8.04H9.05c-4.43,0-8.04-3.6-8.04-8.04V9.96C1.02,5.53,4.62,1.92,9.05,1.92h135.49M17.09,236.84h182.71V81.24h-55.21c-4.43,0-8.04-3.6-8.04-8.04V17.99H17.09v218.85M152.62,65.17h35.81l-1.71-1.71-32.4-32.4-1.71-1.71v35.81M144.58.92H9.05C4.06.92.02,4.97.02,9.96v234.92c0,4.99,4.04,9.04,9.04,9.04h198.78c4.99,0,9.04-4.04,9.04-9.04V73.21c0-.19-.1-.37-.11-.56-.05-.8-.21-1.58-.47-2.34-.1-.29-.17-.56-.3-.84-.44-.96-.99-1.87-1.77-2.65L150.97,3.57c-.78-.78-1.69-1.33-2.65-1.77-.27-.12-.55-.2-.84-.3-.76-.26-1.54-.42-2.35-.47-.19-.01-.36-.11-.56-.11h0ZM18.09,235.84V18.99h117.46v54.21c0,4.99,4.04,9.04,9.04,9.04h54.21v153.6H18.09ZM153.62,64.17V31.77l32.4,32.4h-32.4Z"
    />
    <path
      d="m45.19,108.85c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
      fill="gainsboro"
    />
    <path
      d="m171.69,92.28c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z"
    />
    <path
      d="m45.19,63.67c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h63.25c4.71,0,8.53,3.83,8.53,8.54s-3.83,8.54-8.53,8.54h-63.25Z"
      fill="gainsboro"
    />
    <path
      d="m108.44,47.1c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04h-63.25c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h63.25m0-1h-63.25c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h63.25c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z"
    />
    <path
      d="m45.19,154.02c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
      fill="gainsboro"
    />
    <path
      d="m171.69,137.45c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z"
    />
    <path
      d="m45.19,199.2c-4.71,0-8.54-3.83-8.54-8.54s3.83-8.54,8.54-8.54h126.49c4.71,0,8.54,3.83,8.54,8.54s-3.83,8.54-8.54,8.54H45.19Z"
      fill="gainsboro"
    />
    <path
      d="m171.69,182.63c4.43,0,8.04,3.6,8.04,8.04s-3.6,8.04-8.04,8.04H45.19c-4.43,0-8.04-3.6-8.04-8.04s3.6-8.04,8.04-8.04h126.49m0-1H45.19c-4.99,0-9.04,4.04-9.04,9.04s4.04,9.04,9.04,9.04h126.49c4.99,0,9.04-4.04,9.04-9.04s-4.04-9.04-9.04-9.04h0Z"
    />
  </svg>
</div>
`;

exports[`HamburgerIcon should render correctly 1`] = `
<div>
  <svg
    height="50"
    viewBox="0 0 50 50"
    width="50"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M 5 8 A 2.0002 2.0002 0 1 0 5 12 L 45 12 A 2.0002 2.0002 0 1 0 45 8 L 5 8 z M 5 23 A 2.0002 2.0002 0 1 0 5 27 L 45 27 A 2.0002 2.0002 0 1 0 45 23 L 5 23 z M 5 38 A 2.0002 2.0002 0 1 0 5 42 L 45 42 A 2.0002 2.0002 0 1 0 45 38 L 5 38 z"
      fill="gainsboro"
    />
  </svg>
</div>
`;

exports[`StonksIcon should render correctly 1`] = `
<div>
  <svg
    height="16"
    viewBox="0 0 243.95 250.24"
    width="16"
  >
    <path
      d="m54.21,141.82H9.04c-4.99,0-9.04,4.04-9.04,9.04v90.35c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04v-90.35c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,90.35h-27.11v-72.28h27.11v72.28Z"
      fill="gainsboro"
    />
    <path
      d="m144.56,96.64h-45.18c-4.99,0-9.04,4.04-9.04,9.04v135.53c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04V105.68c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,135.53h-27.11v-117.46h27.11v117.46Z"
      fill="gainsboro"
    />
    <path
      d="m234.92,51.46h-45.18c-4.99,0-9.04,4.04-9.04,9.04v180.71c0,4.99,4.04,9.04,9.04,9.04h45.18c4.99,0,9.04-4.04,9.04-9.04V60.5c0-4.99-4.04-9.04-9.04-9.04Zm-9.04,180.71h-27.11V69.53h27.11v162.64Z"
      fill="gainsboro"
    />
    <path
      d="m170.62,13.75c-.25-1-.72-1.91-1.27-2.75-.09-.14-.1-.3-.2-.44-.11-.16-.3-.23-.42-.38-.64-.75-1.33-1.46-2.18-1.99-1.03-.62-2.15-1.04-3.32-1.21L118.58.1c-4.91-.74-9.55,2.62-10.31,7.55-.76,4.93,2.62,9.55,7.55,10.31l23.09,3.56L22.55,106.85c-4.02,2.96-4.89,8.61-1.94,12.64,1.77,2.41,4.52,3.69,7.3,3.69,1.85,0,3.72-.56,5.34-1.75l116.36-85.33-3.55,23.08c-.76,4.93,2.62,9.55,7.55,10.31.47.07.94.11,1.39.11,4.39,0,8.23-3.19,8.92-7.66l6.86-44.65c.18-1.16.12-2.36-.16-3.53Z"
      fill="gainsboro"
    />
  </svg>
</div>
`;
